ts-tests/scripts/run-test.sh

```diff
@@ -1,17 +1,15 @@
 #!/bin/bash
 
-set -o pipefail
-
 basedir=$(dirname "$0")
 cd "$basedir"
 
 cd ..
 
 tmpdir=$(mktemp -d /tmp/tmp.XXXXXX)
+tmpdir="${1:-$tmpdir}"
```

Comment:
for consistency maybe just use /tmp as fallback (same as in other files)
```suggestion
tmpdir="${1:-/tmp}"
```